### ISSUE TITLE
Upgrade auto-generated UIA testing to include more API tests

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#10926]
#10886 follow-up

Right now, we only generate tests for `Move()`. That's great! But we could/should easily adapt it to test some other APIs like:
- `ExpandToEnclosingUnit()`
- `MoveEndpointByUnit()`

We may also be able to add tests that make sure we work with clones properly. Like...
- create a clone --> "move" the clone --> clone should not be the same as original
- create a clone --> "move" the clone --> "move" the original the same way --> both should be the same
